Engine Hacks/SkillSystem/Skills/StandaloneSkills/StandaloneSkills.event

```diff
@@ -1,3 +1,8 @@
+#ifndef jumpToHack_r0
```

Comment:
This doesn't work like that. Parametrized macros aren't properly recognized by `#if[n]def`. I suggest reworking the hack to either use a non-r0 variant of jumpToHack, or just not use the macro and instead put the raw hook where it would be used (this avoids having to define the jumpToHack_r0 macro in the middle of nowhere)

Reply:
~~mfw all the PRs with issues were just 7743 fixes copypasted verbatim~~ I replaced the macro with the raw form of it and everything seems to be in order still